src/Appwrite/Utopia/Response/Model/Bucket.php

```diff
@@ -28,25 +28,18 @@ public function __construct()
                 'default' => '',
                 'example' => self::TYPE_DATETIME_EXAMPLE,
             ])
-            ->addRule('$read', [
+            ->addRule('$permissions', [
                 'type' => self::TYPE_STRING,
-                'description' => 'File read permissions.',
+                'description' => 'File permissions.',
```

Comment:
If we already improve permissions. We could add here a link to the permissions page to allow devs to learn more about them.